django_project_base/base/middleware/project_middleware.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+from django.conf import settings
+
+
+def get_project(request) -> Optional[object]:
+    project_header: Optional[int] = request.headers.get('Current-Project') or request.headers.get('current-project')
+    if project_header is not None and project_header != '':
+        return project_header
+    project_url_part: str = settings.PROJECT_DEFINED_URL_PART
+    try:
+        project_info: Optional[str] = next(
+            iter(filter(lambda f: f and project_url_part in f, request.path_info.split('/'))))
```

Comment:
Obstaja standarden API za rabijanje URLja. Magari počakaj, da tiu ga razbije že Django ali DRF in potem samo uporabi request.GET ali request.META?

Reply:
done